2017-01-15-tessel-security-scorecard.md

```diff
@@ -0,0 +1,52 @@
+# Tessel's Security Scorecard
+
+In late October of 2016, an estimated 100,000 Internet-connectied devices were used by a group of hackers to attack DNS services as a distributed denial-of-service (DDoS). These devices, mostly routers, printers, and IP cameras, were infected with malware called Mirai, allowing the hackers to take control of those devices and cause outages for major services, like Twitter, Netflix, Spotify, Airbnb, Reddit, Etsy, SoundCloud and The New York Times. 
+
+Now that it is known how Mirai managed to infect all of these Internet-connected devices, Sparkfun wrote about ["5 Easy Ways to Secure Your IoT Devices"](https://www.sparkfun.com/news/2264). The Tessel team has always been proud of the Tessel 2's out-of-the-box experience, so we wanted show off the board's security scorecard based on Sparkfun's list. 
+
+1. Unplug It
+2. Power Cycle
+3. Change The Default Password
+4. Update Firmware
+5. Disable Universal Plug and Play (UPnP)
+6. (Bonus) Disable Telnet and SSH
+
+## Unplug It
+
+> The best possible safeguard against hackers is to simply not have the device available for them.
+
+You control where to power Tessel and even how to power it. Battery, wall socket, or personal computer, you can unplug Tessel anytime.
+
+## Power Cycle
+
+> Another interesting aspect of some malware like Mirai is that it only lives in volatile memory (e.g., RAM). That means simply turning off the device and turning it back on again will rid it of the malware
+
+ There is a handy `t2-cli` command for doing this: `t2 reboot`
```

Comment:
Again, can there be a more direct connection to security? E.g. "When you deploy your project to T2's Flash memory, consider including a periodic auto-reboot to clear anything that may have been introduced to RAM"